### PR TITLE
More, Better, and Fixed Progress Indicators

### LINKED ISSUE #29: Give user idea of progress while pulling oci layers
Just showing layer hashes doesn't give the user any idea of actual progress of said pull. Possible solutions are counting bytes in directory layers are being downloaded to or even just a counter of layers downloaded vs total layers.

### PR DESCRIPTION
**Pull request by corang**
closes #29 #28 
co-author @Michael-Kruggel

### CODE REVIEW

**Comment by corang** [comment]
Bundle creation now:
```sh
corang@halley-server:~/swf/uds-package-software-factory$ devuds bundle create --confirm

 NOTE  Saving log file to /tmp/uds-2023-09-22-23-06-09-1458602760.log

 WARNING  Greetings 🦄! I noticed you used the `uds bundle create [DIRECTORY]` syntax! This syntax
          will be deprecated in favor of `uds create [DIRECTORY]` in an upcoming release

                                                                                                      
  🎁 BUNDLE DEFINITION                                                                                
                                                                                                      


~~package yaml~~

━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✔  Bundle Validated                                                                                                                                                              

                                                                                                      
  🐕 Fetching Packages                                                                                
                                                                                                      

  ✔  Fetched package: init                                                                                                                                                         
  ✔  Fetched package: software-factory-namespaces                                                                                                                                  
  ✔  Fetched package: software-factory-idam-realm                                                                                                                                  
  ✔  Fetched package: software-factory-idam-gitlab                                                                                                                                 
  ✔  Fetched package: software-factory-idam-sonarqube                                                                                                                              
  ✔  Fetched package: gitlab-redis                                                                                                                                                 
  ✔  Fetched package: gitlab-minio                                                                                                                                                 
  ✔  Fetched package: gitlab-postgres                                                                                                                                              
  ✔  Fetched package: gitlab-runner-rbac                                                                                                                                           
  ✔  Fetched package: gitlab-runner                                                                                                                                                
  ✔  Fetched package: sonarqube-postgres                                                                                                                                           
  ✔  Fetched package: sonarqube                                                                                                                                                    
  ✔  Fetched package: software-factory-idam-dns                                                                                                                                    

                                                                                                      
  🚧 Building Bundle                                                                                  
                                                                                                      

    Creating bundle archive ██████████████████████████████████████████████████████████████  67% | 3s
```

**Comment by corang** [comment]
image
image

File: src/pkg/bundle/bundle.go [hunk 1]
```diff
@@ -28,6 +29,8 @@ import (
 
 // Create creates the bundle and outputs to a local tarball
 func Create(b *Bundler, signature []byte) error {
+	message.HeaderInfof("🐕 Fetching Packages")
```
Comment (mikevanhemert) [inline src/pkg/bundle/bundle.go:32]: ❤️

**Review by mikevanhemert — APPROVED** [review]
Awesome enhancement!

**Pull request merged**